entrypoint.sh

```diff
@@ -0,0 +1 @@
+poetry run gunicorn "todo_app.app:create_app()" -b 0.0.0.0:$PORT
```

Comment:
Whenever we have a script file like this, it's a good habit to get into to add [a shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) to the top to help specify what sort of script it is, e.g.
```suggestion
#!/bin/bash
poetry run gunicorn "todo_app.app:create_app()" -b 0.0.0.0:$PORT
```